### ISSUE TITLE
DuplicateRow or InsertRows in table,and quick analysis tool will lost 

### ISSUE THREAD

**Issue by DehaiWang** [qax-os/excelize#1980]
**Description**

i insert some rows in the table zone , and quick analysis tool will lost 

**Steps to reproduce the issue:**
```
package main

import (
	"github.com/xuri/excelize/v2"
	"log"
)

func main() {

	log.SetFlags(log.Lshortfile | log.LstdFlags)

	openFile, err := excelize.OpenFile("aa.xlsx")
	defer openFile.Close()

	if err != nil {
		log.Panicln(err)
		return
	}

	sheetList := openFile.GetSheetList()
	sheetName := sheetList[0]

	rowIndex := 4

	tableList, err := openFile.GetTables(sheetName)
	if err != nil {
		log.Panicln(err)
		return
	}

	tableItem := tableList[0]
	log.Printf("%+v\n", tableItem)

	err = openFile.DuplicateRow(sheetName, rowIndex)
	if err != nil {
		log.Panicln(err)
		return
	}
	
	//nowTime := time.Now().Format(time.DateTime)
	//openFile.SetCellValue(sheetName, "A5", nowTime)
	openFile.Save()

}

```

**Describe the results you received:**
![image](https://github.com/user-attachments/assets/8acae43b-51e7-4a32-824a-bb3ded061a8c)
and then ,quick analysis tool  losted 

**Describe the results you expected:**
![image](https://github.com/user-attachments/assets/7f79d2aa-5a61-4b43-8361-e52a20a2d103)

**Comment by xuri** [comment]
Thanks for your issue. Which version of the Excelize are you using? Could you provide the workbook attachment without confidential info?

**Comment by DehaiWang** [comment]
example workbook
[aa.xlsx](https://github.com/user-attachments/files/16785967/aa.xlsx)

![image](https://github.com/user-attachments/assets/844c5ed7-95db-4235-9446-206df9c51fc7)

**Comment by DehaiWang** [comment]
@xuri  excuse me, do have a plan to fix this ? or have other function to do this ?

**Comment by xuri** [comment]
Yeah, I need some time to fix this, this is caused by incorrect formula adjust for shared formula. Contributions are welcome. I'll certainly accept that patch if somebody did that.

**Comment by pgalbavy-itrs** [comment]
In a related way, is there any chance any changes that address this could also "move" conditional formatting on InsertRows() ?

**Comment by mengpromax** [comment]
It seems that the quick analysis tool for this summary row is a capability provided by WPS (xmlns seems from wps.cn), not a standard capability specified in the OpenXML document. Is it also not very standard if it is supported in Excelize?
